firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/HarmBlockMethod.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.firebase.vertexai.type
+
+/**
+ * Specify if the threshold is used for probability or severity score. If not specified, the
+ * threshold is used for probability score.
+ */
+enum class HarmBlockMethod {
+  SEVERITY,
+  PROBABILITY,
+}
```

Comment:
This description sounds more akin to it's use-site than it's self, no?

```suggestion
/**
 * A category for which the message was blocked.
 */
enum class HarmBlockMethod {
  /**
   * TODO(): explain what a SEVERITY block means
   */ 
  SEVERITY,
  /**
   * TODO(): explain what a PROBABILITY block means
   */ 
  PROBABILITY,
}
```

Reply:
Addressed